### DISCUSSION TITLE
No such file or directory error

### DISCUSSION THREAD

**Discussion by malwux** [TomSchimansky/CustomTkinter#769] [Q&A]
how do i solve this error? when compiling and running the exe I get this huge error that I can't fix

screenshot 1: https://user-images.githubusercontent.com/98716224/205740319-1e77f75b-2399-4336-9435-5cc00d3eea20.png
screenshot 2: https://user-images.githubusercontent.com/98716224/205740350-4092e996-6a4c-4f35-9dfd-60b6cd2d19b3.png

**Comment by dipeshSam** [comment]
Did you set the theme manually in your code?
